components/emotionalRegulation/L.tsx

```diff
@@ -0,0 +1,128 @@
+import { View, Text, TextInput, StyleSheet } from 'react-native';
+import React from 'react';
+import { ACCENT, BLACK, SIZES, WHITE } from '../../styles';
+
+export type props = {
+  days: string;
+  hours: string;
+  mins: string;
+};
+export default function L({
+  time,
+  deadline,
+  handleInput,
+  handleTextInput,
+}: {
+  time: props;
+  deadline: string;
+  handleInput: (v: string, val: string) => void;
+  handleTextInput: (v: string, val: string) => void;
+}) {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.heading}>
+        <Text style={styles.accent}>L</Text>
+        änge schätzen:
+      </Text>
+      <Text style={styles.description}>a) Wie lange brauche ich?</Text>
+      <View style={styles.txtContainer}>
+        <View style={{ alignItems: 'center' }}>
+          <TextInput
+            accessibilityLabel="Trage hier geschätzen Anzahl an Tagen ein"
+            onChangeText={(v) => handleInput('days', v)}
+            placeholder="00"
+            placeholderTextColor="#4F4F4F"
+            style={styles.input}
+            value={time.days}
+          />
```

Comment:
Beim Schritt B kann man für die Frage `Wie lange brauche ich?` auch Symbole eintragen die nicht Zahlen sind, z.B. negative Zahlen :) Eine Idee wäre ein Number-Pad als Tastatur zu setzen.
```suggestion
          <TextInput
            accessibilityLabel="Trage hier geschätzen Anzahl an Tagen ein"
            keyboardType='numeric'
            onChangeText={(v) => handleInput('days', v)}
            placeholder="00"
            placeholderTextColor="#4F4F4F"
            style={styles.input}
            value={time.days}
          />
```

Reply:
Habe ich gleich übernommen :) Ich denke aber mal, dadurch das der Inout den der User da verfasst nicht weiter verwendet wird, ist es ~relativ egal was da rein kommt.

Reply:
Top. Das stimmt der Input ist nicht wirklich sensibel.